modules/accredited_representative_portal/config/initializers/bypass_olive_branch.rb

```diff
@@ -26,20 +26,11 @@ def call(env)
     private
 
     ARP_PATH_INFO_PREFIX = '/accredited_representative_portal'
+    PATH_NORMALIZER = ::ActionDispatch::Journey::Router::Utils
```

Comment:
`Rack::Attack::PathNormalizer` literally resolves to this when this constant is defined. Makes perfect sense to rely on our _request router itself_ for path normalization :chefs_kiss: (this is what Rails uses for routing).